docs/_static/spec/rasa.yml

```
@@ -98,6 +98,10 @@ paths:
                     type: string
                     description: Path of the loaded model
                     example: 20190429-103105.tar.gz
+                  n_training_jobs:
```

optional: I don't really like abbreviating numbers and hence, I'd rather spell it out.
```suggestion
                  number_training_jobs:
```

---

or even>
```suggestion
                  number_of_running_model_trainings:
```

---

What's your opinion on the naming?

---

I'd usually agree with you, but I feel like we'd lose out on the terseness by using `number`. It also seems quite obvious to me that what it's referring to.

---

But `number_training_jobs` is still pretty short, isn't it? I'd at least call it `nr_training_jobs`